### PR TITLE
CGP-0039: Set block target density to 0.8

### PR DESCRIPTION
**Pull request by mcortesi**
First draft of CGP 0039.

This CGP propose modifying the target density from 0.5 to 0.8.

### CODE REVIEW

File: CGPs/cgp-0039.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+date-created: 2021-09-06
+author: Mariano Cortesi (@mcortesi)
+status: DRAFT
+date-executed:
+governance-proposal-id:
+---
+
+## Overview
+
+CGP - Celo Governance Proposal
```
Comment (YazzyYaz) [inline CGPs/cgp-0039.md:13]: Delete this line and add a simple summary of this proposal.

File: CGPs/cgp-0039.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+Pick a four digit number subsequent to the latest PR raised and name your file that with the suffic P (for pending).
+Eg. if the latest PR to the CGP folder was [0034], then your PR should be [0035P].
```
Comment (YazzyYaz) [inline CGPs/cgp-0039.md:16]: please delete lines 15 and 16, see note about adding a simple overview summary of this proposal.

File: CGPs/cgp-0039/cgp_0036.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+[
```
Comment (YazzyYaz) [inline CGPs/cgp-0039/cgp_0036.json:1]: the file is called cgp_0036.json but it's targeting cgp-0039. Is that the right number?

Comment (mcortesi) [reply CGPs/cgp-0039/cgp_0036.json:1]: typo, thanks

**Review by YazzyYaz — APPROVED** [review]

**Pull request merged**